operator/node.go

```diff
@@ -39,35 +45,54 @@ type Options struct {
 	ValidatorOptions validator.ControllerOptions `yaml:"ValidatorOptions"`
 
 	ForkVersion forksprotocol.ForkVersion
+
+	WS        api.WebSocketServer
+	WsAPIPort int
 }
 
 // operatorNode implements Node interface
 type operatorNode struct {
-	ethNetwork     beaconprotocol.Network
-	context        context.Context
-	validatorsCtrl validator.Controller
-	logger         *zap.Logger
-	beacon         beaconprotocol.Beacon
-	net            network.P2PNetwork
-	storage        Storage
-	eth1Client     eth1.Client
-	dutyCtrl       duties.DutyController
+	ethNetwork       beaconprotocol.Network
+	context          context.Context
+	validatorsCtrl   validator.Controller
+	logger           *zap.Logger
+	beacon           beaconprotocol.Beacon
+	net              network.P2PNetwork
+	storage          storage.Storage
+	validatorStorage validator.ICollection
+	qbftStorage      qbftstorageprotocol.QBFTStore
+	eth1Client       eth1.Client
+	dutyCtrl         duties.DutyController
 	//fork           *forks.Forker
 
 	forkVersion forksprotocol.ForkVersion
+
+	ws        api.WebSocketServer
+	wsAPIPort int
 }
 
 // New is the constructor of operatorNode
 func New(opts Options) Node {
+	qbftStorage := qbftstorage.New(opts.DB, opts.Logger, "attestation", opts.ForkVersion)
```

Comment:
lets change "attestation" to `message.RoleTypeAttester.String()`

Reply:
@nivBlox done